volatility3/framework/symbols/metadata.py

```diff
@@ -86,9 +85,21 @@ def pdb_age(self) -> Optional[int]:
         return self._json_data.get("pdb", {}).get("age", None)
 
 
-class LinuxMetadata(interfaces.symbols.MetadataInterface):
+class DwarfMetadata(interfaces.symbols.MetadataInterface):
```

Comment:
Is this specifically DWARF based?  I'm just thinking if we've got btf2json in the pipeline, will it look the same or not?

Reply:
I know, I later changed that. I think you reviewed and older version of it? See my comment on 77778ee